code/game/objects/items.dm

```diff
@@ -129,6 +129,16 @@ GLOBAL_DATUM_INIT(welding_sparks_prepdoor, /mutable_appearance, mutable_appearan
 
 	var/active = FALSE
 
+	//Trick vars
+	///Sound played when you spin the item like the cool dude you are
+	var/spin_sound = 'sound/effects/spin.ogg'
+	///Sound played when you catch your item after a cool trick
+	var/thud_sound = 'sound/effects/thud.ogg'
```

Comment:
You can probably inline thesse vars so they dont have to be on item

Reply:
done